examples/multi_window.py

```diff
@@ -7,6 +7,8 @@
 """
 
 def load_html():
+    import time
+    time.sleep(1)
```

Comment:
There is a semaphor which is set when webview is loaded, so timeout should not be needed. Not sure if the implementation is broken or not, though